### PR TITLE
[sg_wrapper] test if key is basestring

### PR DESCRIPTION
**Pull request by aguiot**

### CODE REVIEW

File: sg_wrapper.py [hunk 1]
```diff
@@ -297,9 +297,9 @@ def find_entity(self, entityType, key = None, find_one = True, fields = None,
                 thisEntityFields = e['fields']
 
         if key:
-            if type(key) == int:
+            if isinstance(key, int):
```
Comment (cchampet) [inline sg_wrapper.py]: Mode "ceinture et bretelles":
```python
if isinstance(key, (int, long)):
  pass
```

**Comment by aoblet** [comment]
For info, putting an unicode string as `filters` directly in the shotgun api is working because `json.dumps` is called in [call_rpc](https://github.com/shotgunsoftware/python-api/blob/master/shotgun_api3/shotgun.py#L3065) method which is doing encoding by default to `utf-8`

**Comment by aoblet** [comment]
For info, putting an unicode string as filters directly in the shotgun api is working because `json.dumps` is called in [call_rpc](https://github.com/shotgunsoftware/python-api/blob/master/shotgun_api3/shotgun.py#L3065) method which is doing encoding by default to `utf-8`

**Comment by aoblet** [comment]
LGTM, this is working, thanks.

File: sg_wrapper.py (hunk 1 above)
Comment (aguiot) [reply sg_wrapper.py]: Le jour ou vous aurez un id superieur a 9223372036854775807, je veux voir ca! ;)

**Pull request merged**